code/modules/mob/living/carbon/crit.dm

```diff
@@ -0,0 +1,65 @@
+/mob/living/carbon/handle_crit()
+
+	/////////////////////////////////////////////
+	//// cogwerks - critical health rewrite /////
+	/////////////////////////////////////////////
+	//// goal: make crit a medical emergency ////
+	//// instead of game over black screen time /
+	/////////////////////////////////////////////
+	if(getBrainLoss() >= 100 && health < 0)
+		Weaken(1)
+		losebreath++
+
+	if(getBrainLoss() >= 120) // braindeath, the real killer
+		visible_message("<span class = 'danger'>[src]'s expression becomes completely blank, a lifeless look in their eyes.</span>")
+		death()
+		return
+
```

Comment:
Here's where the probability check to die, if your health is less than -100, should go.